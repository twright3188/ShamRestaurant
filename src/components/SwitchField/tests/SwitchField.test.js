import React from 'react';
import renderer from 'react-test-renderer';
import { SwitchField } from '../index';

it('switch field render', () => {
    const tree = renderer.create(<SwitchField  text='test' value={false} onChange={() => {}}/>).toJSON();
    expect(tree).toMatchSnapshot();
});