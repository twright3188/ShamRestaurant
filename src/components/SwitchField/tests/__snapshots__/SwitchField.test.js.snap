// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`switch field render 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "borderBottomColor": "rgba(34, 38, 36, 0.15)",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingHorizontal": 15,
        "paddingVertical": 16,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "flexWrap": "wrap",
          "fontFamily": "Helvetica",
          "fontWeight": "400",
        },
        Object {
          "color": "#222624",
          "fontSize": 16,
          "lineHeight": 24,
        },
        Object {
          "width": "80%",
        },
      ]
    }
  >
    test
  </Text>
  <RCTSwitch
    accessibilityRole="button"
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onTintColor="#399AE5"
    style={
      Array [
        Object {
          "height": 31,
          "width": 51,
        },
        Array [
          Object {
            "borderColor": "#C8CCC9",
            "borderWidth": 1.5,
          },
          Object {
            "backgroundColor": "#fff",
            "borderRadius": 16,
          },
        ],
      ]
    }
    thumbTintColor=""
    tintColor="#fff"
    value={false}
  />
</View>
`;
